nixos/modules/services/databases/victorialogs.nix

```
@@ -0,0 +1,114 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+with lib;
+let
```

```suggestion
let
  inherit (lib)
    getBin
    hasPrefix
    literalExpression
    mkBefore
    mkEnableOption
    mkIf
    mkOption
    mkPackageOption
    optionalString
    types
    ;
```
Bit of a nit, but `with lib`; is generally recommend against nowadays for new modules